src/pages/technologies/react.md

```diff
@@ -151,27 +160,51 @@ Screenshots
 
 - [Jack Herrington - Mastering React Batch Updating](https://www.youtube.com/watch?v=MlDTHzK1vKI)
 - [React is slow, what now?](https://nosleepjavascript.com/react-performance/)
-- [Web Dev Junkie - How to implement the MVC pattern in React (very interesting approach to this problem)](https://www.youtube.com/watch?v=LIlTrgVz4QA)
 - [Lydia Hallie - Advanced Rendering Patterns (what shortcuts like ISR, SSG and SSR really mean? )](https://www.youtube.com/watch?v=PN1HgvAOmi8)
+- [Optimizing performance in a React application](https://blog.logrocket.com/5-react-performance-optimization-techniques/)
+- [Modus - How to optimize a React.js app bundle performance to load under 3s](https://www.youtube.com/watch?v=j8NJc60H294)
 
-#### About useMemo and useCallback
+### Memoization in React
 
 - [Kent C. Dodds - Memoization and React](https://epicreact.dev/memoization-and-react/)
+- [Ben Awad - How to use memo in React to Optimize Renders](https://www.youtube.com/watch?v=o-alRbk_zP0)
+
+#### useCallback and useMemo
```

Comment:
starting a section heading with a small letter looks bad

Reply:
I don't think it's that bad. Changing the title to "UseCallback and useMemo" could be confusing.

Reply:
Maybe you could figure out if something can be added before